src/calculateRentalCost.js

```diff
@@ -5,6 +5,13 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
```

Comment:
Replace the hard-coded value `40` with a constant to represent the daily rental cost. This makes the code more maintainable and easier to understand.